tests/plugins/colorbutton/manual/colorhistory/5-tworows.md

```diff
@@ -0,0 +1,23 @@
+@bender-tags: 4.15.0, feature, 1795
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, colorbutton, colordialog, sourcearea, removeformat, undo
+
+1. Click on color button.
+
+1. Pick any color that isn't in color history row yet (e.g. <span style="color:#F39C12">orange</span>).
+
+1. Click on color button.
+
+  **Expected:**
+
+  * Second color history row appeared.
```

Comment:
Maybe we could also extend this test checking if the second row is correctly limited i.e. we are unable to create the third row?

Reply:
Don't you think this case is already covered in `toomanycolors` test?